bin/make_reference_catalog.py

```diff
@@ -21,20 +26,28 @@ class TwinklesReference(InstanceCatalog, AstrometryStars, PhotometryStars):
     parser.add_argument('opsimDB', help='OpSim database sqlite file')
     parser.add_argument('-o', '--outfile', type=str, default='twinkles_ref.txt',
                         help='Filename of output reference catalog')
+    parser.add_argument('--host', type=str,
+                        default='fatboy.phys.washington.edu',
+                        help='CatSim db host name')
+    parser.add_argument('--port', type=int, default=1433, help='db host port')
     args = parser.parse_args()
 
     # you need to provide ObservationMetaDataGenerator with the connection
     # string to an OpSim output database.  This is the connection string
     # to a test database that comes when you install CatSim.
     generator = ObservationMetaDataGenerator(database=args.opsimDB,
                                              driver='sqlite')
-    obsMetaDataResults = generator.getObservationMetaData(fieldRA=(53, 54), fieldDec=(-29, -27), boundLength=0.3)
+    obsMetaDataResults = generator.getObservationMetaData(fieldRA=(53, 54),
+                                                          fieldDec=(-29, -27),
+                                                          boundLength=0.3)
+
+    stars = CatalogDBObject.from_objid('allstars', host=args.host,
+                                       port=args.port)
 
-    # First get the reference catalog
-    stars = CatalogDBObject.from_objid('allstars')
     while True:
```

Comment:
It would be good to document that this is a hack to retry in the case of db connection failure.  I hope there is a better way to deal with this.